site/content/en/docs/Guides/metrics.md

```diff
@@ -60,6 +60,7 @@ Follow the [Stackdriver Installation steps](#stackdriver-installation) to see yo
 | agones_fleet_autoscalers_limited                | The fleet autoscaler is capped (1)                                  | gauge     |
 | agones_gameservers_node_count                   | The distribution of gameservers per node                            | histogram |
 | agones_nodes_count                              | The count of nodes empty and with gameservers                       | gauge     |
+| agones_gameservers_state_duration  | The distribution of gameserver state duration in seconds. Note: this metric could have some missing samples by design. Do not use the `_total` counter the real value for state changes.     | histogram     |
```

Comment:
This should probably be wrapped in a feature shortcode, yeah?

Reply:
Yes, will update this.